MekHQ/src/mekhq/gui/dialog/CustomizePersonDialog.java

```diff
@@ -1159,8 +1157,8 @@ private void addGroup(IOptionGroup group, GridBagLayout gridBag, GridBagConstrai
         panOptions.add(groupLabel);
     }
 
-    private void addOption(IOption option, GridBagLayout gridBag, GridBagConstraints c, boolean editable) {
-        DialogOptionComponent optionComp = new DialogOptionComponent(this, option, editable);
+    private void addOption(IOption option, GridBagLayout gridBag, GridBagConstraints c) {
+        DialogOptionComponent optionComp = new DialogOptionComponent(this, option, true);
```

Comment:
I assume the editable parameter was always passed in as true?

Reply:
Yup